projects/common/coraz7s/coraz7s_system_bd.tcl

```diff
@@ -142,7 +141,4 @@ ad_connect  sys_concat_intc/In0   GND
 
 # interconnects and address mapping
 
-ad_cpu_interconnect 0x45000000 axi_sysid_0
-
-ad_mem_hp0_interconnect $sys_cpu_clk sys_ps7/S_AXI_HP1
-
+ad_cpu_interconnect 0x45000000 axi_sysid_0
```

Comment:
Try to remove that character. I think it is caused by editing files with some windows editor.
Opening and closing the file in vim should solve the problem.

Reply:
Done